### DISCUSSION TITLE
Data aren't showing here

### DISCUSSION THREAD

**Discussion by nicitaacom** [pmndrs/zustand#1870] [Q&A]
https://youtu.be/fZPgBnL2x-Q?t=1503

![image](https://github.com/pmndrs/zustand/assets/39565703/162d86ca-6017-4e01-bfc7-f439848823ff)
![image](https://github.com/pmndrs/zustand/assets/39565703/912be18b-3e14-4bec-be88-9a04185a7f82)
![image](https://github.com/pmndrs/zustand/assets/39565703/f081e787-2b55-4fa1-ac45-ec013c45f66e)

**Comment by dbritto-dev** [comment]
@Nicitaa would you mind sharing us a basic example of you implementation on codesandbox?
